users.tsx

```diff
@@ -1,11 +1,34 @@
-import React from 'react';
-import { FlatList, SafeAreaView, ScrollView, StatusBar, Text, useColorScheme, View } from 'react-native';
+import { gql, useQuery } from '@apollo/client';
+import React, { useEffect } from 'react';
+import { FlatList, SafeAreaView, StatusBar, Text, useColorScheme, View } from 'react-native';
 import { Colors } from 'react-native/Libraries/NewAppScreen';
 import { userList } from './mock';
-import { styles } from './users-style';
 import { UserItem } from './users-model';
+import { styles } from './users-style';
 
 const Users = () => {
+  const listUsersQuerry = gql`
+    query ListUsers($pageInfo: PageInputType) {
+      users(pageInfo: $pageInfo) {
+        nodes {
+          id
+          email
+          role
+        }
+      }
+    }
+  `;
+  const listUsersData = {
+    variables: {
+      pageInfo: {
+        offset: 0,
+        limit: 10,
+      },
+    },
+  };
+  const { data } = useQuery(listUsersQuerry, listUsersData);
+
+  console.log(data);
```

Comment:
Vou dividir as responsabilidades seguindo SOLID quando for fazer o rebase. Isso pra poder seguir a estrutura já existente do outro pr

Reply:
Vc pode adiantar os rebases pra antes do CR. Assim a gente já revisa o código final.

Reply:
Boa, vou fazer isso

Reply:
feito.